sympy/series/formal.py

```diff
@@ -824,8 +832,8 @@ def _compute_fps(f, x, x0, dir, hyper, order, rational, full):
         return None
 
     ak = sequence(result[0], (k, result[2], oo))
-    xk = sequence(x**k, (k, 0, oo))
-    ind = result[1]
+    xk = sequence(x**k * symb, (k, 0, oo))
```

Comment:
Probably if you do a ``powsimp`` here 
```
xk_formula = powsimp(x**k * symb)
xk = sequence(xk_formula, (k, 0, oo))
```

This is will make it more easier to read as well. Probably you might be able to drop some ``powsimp`` calls.